api/app/Http/Middleware/TransformInputDataMiddleware.php

```diff
@@ -27,8 +27,11 @@ public function handle($request, Closure $next)
             // Find any potential camelCase keys in the 'root' array, and convert
             // them to snake_case
             if (!ctype_lower($key)) {
-                $request->offsetSet(snake_case($key), $value);
-                $request->offsetUnset($key);
+                // Only convert if the key will change
+                if ($key != snake_case($key)) {
```

Comment:
Out of interest, what is an example of a string that would not enter this block?